### PR TITLE
Zib-ApgarScore

### PR DESCRIPTION
**Pull request by Ashwin-nictiz**
TODO: 

* Zib compliance error

### CODE REVIEW

File: resources/zib/zib-ApgarScore.10Minute.xml [hunk 1]
```diff
@@ -0,0 +1,283 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="zib-ApgarScore.10Minute" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/zib-ApgarScore.10Minute" />
```
Comment (LuudSlagter) [inline resources/zib/zib-ApgarScore.10Minute.xml]: Based on the profiling guidelines this profile should be called _zib-ApgarScore-10Minute_ (the same holds for the other profiles). See section 12.1.1.1 of the profiling guidelines: _If there are multiple implementations of the same building block for different uses, a use specific suffix is appended._

**Review by LuudSlagter — CHANGES_REQUESTED** [review]

File: resources/nl-core/nl-core-ApgarScore.10Minute.xml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  <purpose value="A derived profile from [zib-ApgarScore.10Minute](http://nictiz.nl/fhir/StructureDefinition/zib-ApgarScore.10Minute) to provide a version better suited for implementation purposes. This profile augments the base profile with elements found in the various use cases that have adopted the zib." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <kind value="resource" />
+  <abstract value="false" />
+  <type value="Observation" />
+  <baseDefinition value="http://nictiz.nl/fhir/StructureDefinition/zib-ApgarScore.10Minute" />
+  <derivation value="constraint" />
+  <differential>
```
[the remark this answers is not in the document]
Comment (Ashwin-nictiz) [reply resources/nl-core/nl-core-ApgarScore.10Minute.xml]: Done

File: resources/zib/zib-ApgarScore.10Minute.xml [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+      <path value="Observation.value[x]" />
+      <slicing>
+        <discriminator>
+          <type value="type" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.value[x]:valueInteger">
```
[the remark this answers is not in the document]
Comment (Ashwin-nictiz) [reply resources/zib/zib-ApgarScore.10Minute.xml]: Fix

File: resources/zib/zib-ApgarScore.10Minute.xml (hunk 1 above)
Comment (Ashwin-nictiz) [reply resources/zib/zib-ApgarScore.10Minute.xml]: Done

File: resources/zib/zib-ApgarScore.5Minute.xml [hunk 4]
```diff
@@ -0,0 +1,283 @@
...
+      <path value="Observation.component" />
+      <sliceName value="respiratoryScore" />
+      <max value="1" />
+    </element>
+    <element id="Observation.component:respiratoryScore.code">
+      <path value="Observation.component.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://loinc.org" />
+          <code value="32410-3" />
```
[the remark this answers is not in the document]
Comment (Ashwin-nictiz) [reply resources/zib/zib-ApgarScore.5Minute.xml]: Fix

**Review by LuudSlagter — APPROVED** [review]

**Pull request merged**